conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class MeshlinklibConan(ConanFile):
+    name = "meshlink"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Elearcommonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    #requires="catta/0.1@demo/dev"
```

Comment:
This can be removed since the catta source is being built as part of the meshlink codebase. I also don't think we need to build the catta conan package anymore since we will not be using it anymore.

Reply:
We can remove catta as dependency but we have to create separate package for catta because during linking with dependent library either meshlink or catta is linking

Reply:
If catta is statically linked into meshlink it's not necessary to create a catta package